scripts/pre_commit_linter.py

```diff
@@ -2745,11 +2634,13 @@ def perform_all_lint_checks(self):
 
         common_messages = super(
             JsTsLintChecksManager, self).perform_all_lint_checks()
+        arg1 = (self.js_filepaths, _STDOUT_LIST, self.process_manager)
+        arg2 = (self.all_filepaths, _STDOUT_LIST, self.process_manager)
 
         super(JsTsLintChecksManager, self)._run_multiple_checks(
-            self._check_extra_js_files,
-            self._check_js_and_ts_component_name_and_count,
-            self._check_directive_scope
+            (self._check_extra_js_files, arg1),
+            (self._check_js_and_ts_component_name_and_count, arg2),
```

Comment:
Do we need to pass all the files to this method?

Reply:
In this context, all files refer to js and ts files.

Reply:
I see, thanks for the explanation. :)